docs/apache-airflow-providers-tabular/redirects.txt

```diff
@@ -0,0 +1,3 @@
+connections.rst ../apache-airflow-providers-apache-iceberg/connections.rst
+changelog.rst ../apache-airflow-providers-apache-iceberg/changelog.rst
+index.rst ../apache-airflow-providers-apache-iceberg/index.rst
```

Comment:
This redirect not working as expected.

Reply:
I think we don't need redirect here.
It's a different provider